compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/text/BasicTextField.desktop.kt

```diff
@@ -133,7 +134,8 @@ fun BasicTextField(
     keyboardOptions: KeyboardOptions = KeyboardOptions.Default,
     keyboardActions: KeyboardActions = KeyboardActions.Default,
     singleLine: Boolean = false,
-    maxLines: Int = Int.MAX_VALUE,
+    maxLines: Int = if (singleLine) 1 else Int.MAX_VALUE,
```

Comment:
Can just be `maxLines: Int = Int.MAX_VALUE` because if `singleLine` it's ignored below anyway.

Reply:
Better to keep the same code as in the original text field, to simplify future copy-pastes